course_content/cartopy_course/solutions/cartopy_exercise_2

```diff
@@ -0,0 +1,24 @@
+
+# Cartopy course, exercise 2
+
+# Use 'set_global' to work out what it was actually doing earlier.
+
+# Code as previously ...
+proj = ccrs.EquidistantConic()
+x0, y0 = -50, -30
+x1, y1 = 10, 55
```

Comment:
The problem with including this preamble is that it when the user loads in this file, they get a bunch of other stuff.

I don't see the point of keeping this as it wouldn't run anyway as you haven't imported cartop/matplotlib

Reply:
Ok, you're right - I was thinking about how it could be made "more standalone", but I stopped short of imports.
It's probably simpler to just keep it short.  Will do that ...